src/Table.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { forwardRef, memo, type ReactNode, type CSSProperties } from "react";
+import { assert } from "tsafe/assert";
+import type { Equals } from "tsafe";
+import { fr } from "./fr";
+import { cx } from "./tools/cx";
+import { symToStr } from "tsafe/symToStr";
+import type { FrClassName } from "./fr/generatedFromCss/classNames";
+
+export type TableProps = {
+    fixed?: boolean;
+    data: ReactNode[][];
+    className?: string;
+    caption?: ReactNode;
+    bordered?: boolean;
+    headers?: ReactNode[];
+    noScroll?: boolean;
```

Comment:
```suggestion
    /** Default: false */
    noScroll?: boolean;
```
Always javadoc the default for optional props.